README.md

```diff
@@ -23,7 +23,7 @@ These instructions will get you a copy of the role for your ansible playbook. On
 
 ### Prerequisities
 
-Ansible 2.4.1.0 version installed.
```

Comment:
It would be nice if min_ansible_version match with version defined in the Readme file: https://github.com/idealista/prometheus_server-role/blob/master/meta/main.yml#L6